include/grpc++/support/auth_context.h

```diff
@@ -72,20 +72,26 @@ class AuthPropertyIterator
   const char* name_;
 };
 
+/// Class encapsulating the Authentication Information.
+/// 
+/// It includes the secure identity of the peer, the type of secure transport
+/// used as well as any other properties required by the authorization layer.
```

Comment:
Probably worthwhile... it might also be worth adding some wrapper methods...

string x509_san_property_name()